searxng/docker-compose.searxng.yml

```diff
@@ -0,0 +1,51 @@
+version: '2'
+
+networks:
+  searxng-internal:
+
+services:
+  searxng:
+    image: searxng/searxng:${SEARXNG_IMAGE_VERSION:-latest}
+    volumes:
+      - './searxng/searxng/:/etc/searxng:rw'
+    networks:
+      - 'srv'
+    healthcheck:
+      test: ["CMD-SHELL", "wget -q --spider --proxy=off localhost:8080/healthz || exit 1"]
+    restart: always
+    networks:
+      - 'srv'
+      - 'searxng-internal'
+    environment:
+      - LIMITER=true
+      - REDIS_URL=redis://searxng-redis:6379/0
+      - IMAGE_PROXY=true
+      - SEARXNG_BASE_URL=https://searx.${SITE:-localhost}/'
+    depends_on:
+      - searxng-redis
+    cap_drop:
+      - ALL
+    cap_add:
+      - CHOWN
+      - SETGID
+      - SETUID
+      - DAC_OVERRIDE
+    labels:
+      - 'traefik.enable=true'
+      - 'traefik.http.routers.searxng.rule=Host(`searx.${SITE:-localhost}`)'
+      - 'traefik.http.services.searxng.loadbalancer.server.port=8080'
+
+  searxng-redis:
```

Comment:
For Redis, I would have used something more like this: https://github.com/tomMoulard/make-my-server/blob/master/mastodon/docker-compose.mastodon.yml#L22-L32

The `command` and `tmpfs` commands might be needed though.